### PR TITLE
 Fix timeout issues

### PR DESCRIPTION
**Pull request by stanislas-m**

### CODE REVIEW

File: plugins/plugins.go [hunk 1]
```diff
@@ -20,8 +20,9 @@ import (
 
 const timeoutEnv = "BUFFALO_PLUGIN_TIMEOUT"
 
+var t = time.Second * 2
+
 func timeout() time.Duration {
-	t := time.Second
```
Comment (mclark4386) [inline plugins/plugins.go:24]: Curious why the change in scope here?

Comment (stanislas-m) [reply plugins/plugins.go:24]: Because of the oncer, the timeout is set once to the `BUFFALO_PLUGIN_TIMEOUT` value. Then for each call to `timeout()` after that, `t` is always set to the default value.

**Review by mclark4386 — APPROVED** [review]

**Comment by mclark4386** [comment]
We should probably also look into why the buffalo-pop available command is taking almost 2 seconds at times ^^;

**Comment by stanislas-m** [comment]
Yeah, sure. `buffalo-plugins` does it too, but that's not the case for `buffalo-docker`, for instance.
